libkbfs/config_local.go

```diff
@@ -261,6 +263,16 @@ func MakeLocalTeams(teams []libkb.NormalizedUsername) []TeamInfo {
 			},
 			LatestKeyGen: FirstValidKeyGen,
 		}
+		// If this is a subteam, set the root ID.
```

Comment:
Correct.  Only the top-level root team has a quota, no matter how deep the subteam tree is.